.circleci/config.yml

```diff
@@ -0,0 +1,54 @@
+version: 2
+jobs:
+
+  build:
+    docker:
+      - image: circleci/openjdk:8-node
```

Comment:
can we use a plain ol' java image for this job? (i.e. no node)

Reply:
So weirdly circleci don't offer a pre-built image with java but no node: https://circleci.com/docs/2.0/circleci-images/

I figured by using the official circleci ones we're more likely to be cached :)